packages/esm-generic-patient-widgets-app/src/obs-table/obs-table.component.tsx

```diff
@@ -48,7 +48,19 @@ const ObsTable: React.FC<ObsTableProps> = ({ patientUuid }) => {
               break;
 
             case 'Number':
-              rowData[obs.conceptUuid] = obs.valueQuantity?.value;
+              let decimalPlaces: number = config.data.find(
```

Comment:
```suggestion
              const decimalPlaces: number | undefined = config.data.find(
```

Reply:
Since we're already parsing the data to add the `dataType` is there any real reason to get the decimal places here instead of just filling it in when we determine the `dataType` is number?